### ISSUE TITLE
OmniglotNShot

### ISSUE THREAD

**Issue by li829** [dragen1860/MAML-Pytorch#37]
I used omniglot to train this model, but I found that train_data and test_data used in fine-tune are same classes. Such as 5-way, train_data are [0,1, 2, 3 ,4] and test_data also are [0, 1, 2, 3, 4]。

**Comment by janbolle** [comment]
They will always be the same.
The ml will learn to map an unlabeled image to a class [0, 1, 2, 3, 4] by finding similar features and not the "label" of a class
